### PR TITLE
Get playwright green

### PR DESCRIPTION
**Pull request by franknoirot**
A number of playwright-electron tests have been testing reliably when run alone with `--repeat-each=50` locally, but [failing in CI like here](https://github.com/KittyCAD/modeling-app/actions/runs/10731890169/job/29762700806?pr=3807#step:20:19553). No behavior change seems to have occurred to break the failing tests.

This seems to be mostly due to run order affecting which panes are open are closed at the start of a test, combined with page actions that don't take into account if a pane is already opened before attempting to open it, and visa versa. I've made some tweaks to see if that gets green more consistently.

### CODE REVIEW

File: e2e/playwright/test-utils.ts [hunk 1]
```diff
@@ -585,6 +585,15 @@ export async function getUtils(page: Page, test_?: typeof test) {
...
 
+    /**
+     * @deprecated Sorry I don't have time to fix this right now, but runs like
+     * the one linked below show me that setting the open panes in this manner is not reliable.
+     * You can either set `openPanes` as a part of the same initScript we run in setupElectron/setup,
+     * or you can imperatively open the panes with functions like {openKclCodePanel}
+     * (or we can make a general openPane function that takes a paneId).,
+     * but having a separate initScript does not seem to work reliably.
+     * @link https://github.com/KittyCAD/modeling-app/actions/runs/10731890169/job/29762700806?pr=3807#step:20:19553
+     */
```
Comment (lf94) [inline e2e/playwright/test-utils.ts:596]: Ah, dang :(

**Review by lf94 — APPROVED** [review]
Looks good to me.

File: e2e/playwright/test-utils.ts (hunk 1 above)
Comment (franknoirot) [reply e2e/playwright/test-utils.ts:596]: Ah yeah I'm sorry I still believe in the spirit of what you're going for so I didn't want to rip it out! I just think it needs to have different implementation details.

Comment (franknoirot) [reply e2e/playwright/test-utils.ts:596]: And I might be off and it's something about the `page.reload()` or something else in that function, I just saw it was a common thread between the tests failing across the PRs.

Comment (lf94) [reply e2e/playwright/test-utils.ts:596]: Yeah yeah all good :heart:

**Review by jtran — APPROVED** [review]

**Comment by franknoirot** [comment]
There's a race in "Closing settings modal should..." that still needs addressed because pasting the code into the editor does not wait for that code to be persisted to the file system, but I'm merging this now

**Pull request merged**